### PR TITLE
migrate ignitus-Introduction to TS

### PR DESCRIPTION
**Pull request by toshihidetagami**
I migrated files in `ignitus-Introduction` folder to Typescript. This is part of #477.

- `reducers.js` seems unused, so I deleted it.
- due to the [issue](https://stackoverflow.com/questions/33977054/cannot-compile-namespaces-when-the-isolatedmodules-flag-is-provided) on `--isolatedModules`, I added dummy export on empty files.

### CODE REVIEW

File: src/ignitus-HomePage/ignitus-Introduction/constants.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export const RESUME: string = 'https://storage.googleapis.com/ignitus_assets/ig-assets/Assets_Resume.svg';
```
[the remark this answers is not in the document]
Comment (toshihidetagami) [reply src/ignitus-HomePage/ignitus-Introduction/constants.ts]: No problem on my local machine. Maybe hound does not recognize Typescript?

**Comment by toshihidetagami** [comment]
@divyanshu-rawat 
I added a commit. Could you check if I understand your suggestion correctly?

File: src/ignitus-HomePage/ignitus-Introduction/constants.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (divyanshu-rawat) [reply src/ignitus-HomePage/ignitus-Introduction/constants.ts]: That's fine Ignore hound in this case :)

**Comment by divyanshu-rawat** [comment]
@toshihidetagami LGTM now 👍

**Comment by toshihidetagami** [comment]
Thanks for merging!

**Pull request merged**